rules/windows/defense_evasion_suspicious_scrobj_load.toml

```diff
@@ -13,27 +13,25 @@ Identifies scrobj.dll loaded into unusual Microsoft processes. This usually mean
 executed in the target process.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.library-*", "logs-windows.sysmon_operational-*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Suspicious Script Object Execution"
 risk_score = 47
 rule_id = "4ed678a9-3a4f-41fb-9fea-f85a6e0a0dff"
 severity = "medium"
 tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Data Source: Elastic Defend"]
+timestamp_override = "event.ingested"
 type = "eql"
 
 query = '''
-sequence by process.entity_id with maxspan=2m
-  [process where host.os.type == "windows" and event.type == "start"
-   and (process.code_signature.subject_name in ("Microsoft Corporation", "Microsoft Windows") and
-   process.code_signature.trusted == true) and
-     not process.executable : (
+any where host.os.type == "windows" and 
+ (event.category == "library" or (event.category == "process" and event.action : "Image loaded*")) and 
+ (dll.name : "scrobj.dll" or file.name : "scrobj.dll") and 
```

Comment:
only problem is that this could cause schema failures due to missing fields. You may need to use `?` optional field syntax to keep it happy across sources

Reply:
adjusted using `?`